tests/Command/UpdateAppPropertyCommandTest.php

```diff
@@ -56,6 +76,51 @@ public function testExecuteFile(): void
         self::assertContains('"longDescription": "Long description', $commandTester->getDisplay());
     }
 
+    public function testExecuteFileJson(): void
+    {
+        $application = new Application();
+        $application->add(new UpdateAppPropertyCommand());
+        $application->add(new GetRepository());
+
+        $command = $application->find('update-app-property');
+
+        $fileName = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('dev-portal-test');
+        file_put_contents($fileName, uniqid('Long description') . "\n\n with newlines");
+        $commandTester = new CommandTester($command);
+        $commandTester->execute([
+            'command'  => $command->getName(),
+            'vendor' => getenv('KBC_DEVELOPERPORTAL_TEST_VENDOR'),
+            'app' => getenv('KBC_DEVELOPERPORTAL_TEST_APP'),
+            'property' => 'longDescription',
+            '--value-from-file' => $fileName,
+        ]);
+        @unlink($fileName);
+        self::assertEquals(0, $commandTester->getStatusCode());
+        self::assertContains('"longDescription": "Long description', $commandTester->getDisplay());
+    }
+
+    public function testExecuteJsonFileInvalid(): void
```

Comment:
Přílepková metoda, nemělo by se to jmenovat `testExecuteInvalidJsonFile`? sám jsem to teda doteď dělal :-)

Reply:
a to je spatne? me to prijde naopak dobry, ze vidis, ze nejaky testy spolu souvisi

Reply:
Tak to by měly, když jsou v jedný testsuitě. A když spolu nesouvisej, tak zvláštní testsuita? Přijde mi, že pokud několik testů sdílí stejný prefix, tak jsou to adepti na vlastní testsuitu (pojmenovanou po prefixu), a uvnitř je to pak mnohem hezčí.

Reply:
prijde mi sice trosku divny to rozdelovat, ale nedokazu prijit na zadny zavazny duvod proc to nerozdelovat, tak jsem to rozdelil :)